features/step_definitions/helper_services.rb

```diff
@@ -183,7 +184,7 @@
       | port_spec | <%= cb.ldap_port %>:389 |
       })
     step %Q/the step should succeed/
-end
+end  
```

Comment:
When you self review, did you notice the diff here? Remove the spaces

Reply:
@xingxingxia as per comment space removed, please review.